### DISCUSSION TITLE
What revset is calculated for `git restack`? It seems to include commits that it shouldn't.

### DISCUSSION THREAD

**Discussion by maciasello** [arxanas/git-branchless#917] [Q&A] [answered]
I've been using `git branchless` for a few days and it works nearly perfectly. :1st_place_medal: 

The issue I'm facing is that `git restack` refuses to work due to too wide revset it detected for restacking.

Here are command outputs:
<img width="843" alt="image" src="https://user-images.githubusercontent.com/2512757/233962798-2c1921f6-e254-48c0-89a5-0b68e981a457.png">
<img width="838" alt="image" src="https://user-images.githubusercontent.com/2512757/233963048-0f1a1a7b-306f-4bbe-abfd-0f1ed5695f30.png">

So basically, the "current stacks" are calculated as expected. Other workflows work super fine and I manage to overcome the missing restack with `move`. Just `restack` seems to be stuck with a lot more commits to affect than it seems reasonable?

**Comment by arxanas** [comment, marked as answer]
Hi @maciasello, I've just changed this in https://github.com/arxanas/git-branchless/pull/914 as per https://github.com/arxanas/git-branchless/discussions/814#discussioncomment-5696753. You can try out the latest development build to see if resolves your issue, or manually run `git restack 'draft()'`.

**Reply by maciasello** [reply]
`git restack 'draft()'` works correctly. Thanks for the answer! I will try to verify the dev build sometime during the day.
